### ISSUE TITLE
[Feature Request]How do i disable downloading for an ordinary account?

### ISSUE THREAD

**Issue by akimolll** [hkalexling/Mango#249]
I have to say that Mango is really great. I run it in Synology Docker. I want to share more comics with more accounts. They are ordinary accounts. I just hope they can watch it online without having to download it. But I can't seem to find such a function.

Will this feature be added in the future?
Thank you for all your efforts! ♥

**Comment by hkalexling** [comment]
Thanks for reaching out! Currently all users can download the CBZ/CBR files locally and there's no way to disable it. I can add a config option to disable it for non-admin users when I have the time. 

I think eventually we will need a fine-grained permission control system where the admins can enable/disable features for user groups or individual users. This would cover #13 and #66.

**Comment by akimolll** [comment]
> Thanks for reaching out! Currently all users can download the CBZ/CBR files locally and there's no way to disable it. I can add a config option to disable it for non-admin users when I have the time.
> 
> I think eventually we will need a fine-grained permission control system where the admins can enable/disable features for user groups or individual users. This would cover #13 and #66.

Yes, it's really great. Permissions need to be subdivided, otherwise everyone is an administrator, and everything will start to be chaotic.
